packages/angular/src/app/pages/planning-task-list/task-list-grid/task-list-grid.component.ts

```diff
@@ -51,7 +50,7 @@ export class TaskListGridComponent implements OnChanges {
   constructor(private router: Router) {
   }
 
-  addRow() { 
+  addRow() {
```

Comment:
it seems this method can be removed

Reply:
It is not connected with my pr. But ok, I will remove